VoltAir/qml/actors/Orb.qml

```diff
@@ -43,6 +43,13 @@ Actor {
         return chosenLetter;
     }
 
+    function langDir() {
+        var l = TR.getCurrentLanguage();
```

Comment:
Well, yes, but you already merged it? I can change it when I add new images to Orb.qml.